OpenRA.Game/Widgets/Widget.cs

```diff
@@ -204,6 +206,8 @@ public Widget(Widget widget)
 			IgnoreChildMouseOver = widget.IgnoreChildMouseOver;
 			IgnoreMouseOver = widget.IgnoreMouseOver;
 
+			ChromeMetrics.Get<string>("DefaultCursor");
```

Comment:
`defaultCursor = `?

Reply:
Actually, this still won't work - this is inside the copy constructor, so won't apply for widgets constructed from yaml.

Instead, assign the value directly where its declared like you've done everywhere else.

Reply:
Yes, that was stupid. Fixed.